pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -190,6 +233,41 @@ func migratableDomXML(dom cli.VirDomain, vmi *v1.VirtualMachineInstance) (string
 	return string(buf.Bytes()), nil
 }
 
+func updateCPUSetAttributeForXMLElement(v *xml.StartElement, cpuSetMapping map[int]int) error {
+	for i := 0; i < len(v.Attr); i++ {
+		if v.Attr[i].Name.Local == "cpuset" {
+			currentCPUsStr := v.Attr[i].Value
+
+			// if a collection of CPUs is defined, replace all CPUs
```

Comment:
I'm probably missing something, but this seems to concatenate the original list and the new one, not replace all CPUs.